airbyte-integrations/connectors/destination-snowflake-cortex/destination_snowflake_cortex/globals.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+"""Globals and constants for the destination."""
+
+from __future__ import annotations
+
+
+DOCUMENT_ID_COLUMN = "document_id"
+CHUNK_ID_COLUMN = "chunk_id"
+METADATA_COLUMN = "metadata"
+DOCUMENT_CONTENT_COLUMN = "document_content"
+EMBEDDING_COLUMN = "embedding"
```

Comment:
Moved to a central 'globals' file so multiple classes can depend on these without risking circular dependencies.